client/trigger_test.go

```diff
@@ -51,10 +51,6 @@ func TestTriggers(t *testing.T) {
 				Value: 10000,
 			},
 			Recipients: []NotificationRecipient{
```

Comment:
This is a hack.. the client tests need to be revisited in general as they're starting to show strain under more frequent usage.